### DISCUSSION TITLE
Deployment Stagnant on Update.

### DISCUSSION THREAD

**Discussion by Bhaney44** [vercel/community#3627] [Help] [answered]
### Summary

Hi,

I really enjoy Vercel for deploying production code. One thing that is really helpful is that the code automatically pushes from the main branch to the website, for near real-time updates. So, if you find an error, such as a typo in the website, it can easily be fixed.

I am running into a problem today however, where I have updated the code on GitHub, but it is not pushing or deploying automatically via Vercel. I checked around to try and discover the error and the [docs](https://vercel.com/docs/concepts/deployments/git#production-branch) suggested one problem was that I was writing to a branch, rather than the main. However, I checked and in fact, I was making edits to the main branch. 

I am deploying a private repo, or I would share the code here. Otherwise, this is the simplest case of a website, there are no extras, add ons, or other considerations. I just want the code to update.

For reference, you can see I recently updated the files today.
<img width="886" alt="Screenshot 2023-08-15 at 1 13 48 PM" src="https://github.com/vercel/community/assets/43055154/a7b173ff-99c1-41fb-99e8-6d21538e692c">

But, that production is over a week old.
<img width="890" alt="Screenshot 2023-08-15 at 1 14 04 PM" src="https://github.com/vercel/community/assets/43055154/bb941640-5ebc-41a0-89db-16b02b2635d2">

Typically, it only takes about 1-3 minutes to update. It also looks like according to the docs, this should have triggered an automatic update because the deployment was to the main branch, which is the production branch.
<img width="1216" alt="Screenshot 2023-08-15 at 1 17 18 PM" src="https://github.com/vercel/community/assets/43055154/dd3f9df9-78c0-49f5-bf47-911cfe1fb676">

So, I am not sure why the update wouldn't happen automatically as it has in the past, or if there is something I need to do, to get Vercel to collect the new code from GitHub. Any help or suggestions would be appreciated. 

Thanks,

Brian

### Steps to Reproduce

1. Deploy website.
2. Update code.

**Comment by leerob** [comment, marked as answer]
Sorry about this, there is an ongoing GitHub incident.

> Due to a problem with GitHub, we are currently experiencing delayed deployments made with Vercel for GitHub. For more information, see the GitHub status: https://www.githubstatus.com/incidents/ljhhhw2tplqp

> In the meantime, you can still use the Vercel CLI to deploy: https://vercel.com/docs/cli

**Reply by Bhaney44** [reply]
You rock. Thank you!
